gromacs/__init__.py

```diff
@@ -242,6 +242,17 @@ def stop_logging():
     log.clear_handlers(logger)  # this _should_ do the job...
 
 
+# Try to load environment variables setted by GMXRC
+gmxrc = config.cfg.get("Gromacs", "GMXRC")
+if not gmxrc:
```

Comment:
@pslacerda , I am sorry, I am a total idiot asking for the change `if gmxrc != ""` to `if not gmxrc:`: This is now doing the *opposite*. Instead it should be
```python
if gmxrc:
```

D'oh! I will fix it.

And come to think: If the user has an old cfg file, the `cfg.get()` will through a `NoOptionError`. But I will fix this by adding Gromacs/GMXRC to the defaults.